draft-ietf-oauth-transaction-tokens.md

```diff
@@ -306,34 +312,47 @@ In the JWT Header:
 ~~~
 {: #figtxtokenheader title="Example: Txn-Token Header"}
 
-## JWT Body {#txn-token-body}
+## JWT Body Claims {#txn-token-claims}
+
+The transaction token body follows the JWT format and includes existing
+JWT claims as well as defines new claims. These claims are described below:
+
+`iss`:
+: OPTIONAL The `iss` claim as defined in {{RFC7519}} is not required as Txn-Tokens are bound to a single trust domain as defined by the `aud` claim and often the signing keys are known. The `iss` claim MAY be used in cases where the signing keys are not predetermined or it is desired that the Txn-Token Service signs with unique keys.
```

Comment:
I would like to make this language a bit more normative. How about something like "The iss claim ... is optional. In situations where there are multiple Txn-Token issuers for the same trust domain, the iss claim MUST be included and MUST describe the issuer of the Txn-Token."

Reply:
@tulshi are you ok if I just change the MAY to MUST?  It is still possible for multiple Txn-Token issues to use a predetermined signing key.

I can add a sentence along the lines of "The signing key MUST be discoverable via the `iss` claim".

If we say that, do we need to define that discovery mechanism? or can we just refer to OAuth AS Metadata?